packages/docs/getting-started/running-a-validator.md

```diff
@@ -117,177 +133,170 @@ In all the commands we are going to see the `CELO_IMAGE` variable to refer to th
 
 ```bash
-screen -ls
+# On the proxy machine, retrieve the proxy enode
+echo $(docker exec celo-proxy geth --exec "admin.nodeInfo['enode'].split('//')[1].split('@')[0]" attach | tr -d '"')
 ```
 
-And re-atach to any of the existing sessions:
+Now we need to set the proxy enode and proxy IP address in environment variables on the validator machine.
 
 ```bash
-screen -r -S celo-validator
+# On the validator machine
+export PROXY_ENODE=<proxy enode>
+export PROXY_IP=<proxy ip address>
 ```
 
-### Reference Script
+Let's connect the validator to the proxy:
 
-You can use (and modify if you want) this [reference bash script](../../../scripts/run-docker-validator-network.sh) automating all the above steps. It requires Docker and screen.
+### Connect the Validator to the Proxy
 
-You can see all the options using the following command:
+When starting up your validator, it will attempt to create a network connection between the validator machine and the proxy machine.  You will need make sure that your proxy machine has the appropriate firewall settings to allow the validator to connect to it.
+
+Specifically, on the proxy machine, port 30303 should allow TCP and UDP connections from all ip addresses.  And port 30503 should allow TCP connections from the IP address of your validator machine.
+
+Once that it completed, go ahead an run the validator.
 
 ```bash
-./run-docker-validator-network.sh help
+# On the validator machine
+docker run -v $PWD/validator:/root/.celo --entrypoint sh --rm $CELO_IMAGE -c "echo VALIDATOR_ADDRESS_PASSWORD > /root/.celo/.password"
+docker run --name celo-validator -d --restart always -p 30303:30303 -p 30303:30303/udp -v $PWD/validator:/root/.celo $CELO_IMAGE --verbosity 3 --networkid $NETWORK_ID --syncmode full --mine --istanbul.blockperiod=5 --istanbul.requesttimeout=3000 --etherbase $CELO_VALIDATOR_ADDRESS --nodiscover --proxy.proxied --proxy.proxyenodeurlpair=enode://$PROXY_ENODE@$PROXY_IP:30503\;enode://$PROXY_ENODE@$PROXY_IP:30303  --unlock=$CELO_VALIDATOR_ADDRESS --password /root/.celo/.password
 ```
 
-## Get elected as validator
+The `mine` flag does not mean the node starts mining blocks, but rather starts trying to participate in the BFT consensus protocol. It cannot do this until it gets elected -- so next we need to stand for election.
 
-### Obtain and lock up some Celo Gold for staking
+The `networkid` parameter value of `1101` indicates we are connecting the Baklava Beta network.
 
-To participate in The Great Celo Stake Off (aka TGCSO) and get fauceted it's necessary to register online via an [online form](https://docs.google.com/forms/d/e/1FAIpQLSfbn5hTJ4UIWpN92-o2qMTUB0UnrFsL0fm97XqGe4VhhN_r5A/viewform).
+### Register the Accounts
 
-### Submitting transactions
+Hopefully by now, you have received Celo Gold, you can finally submit transactions with the `celocli` (remember to unlock the accounts). Let's register our accounts:
```

Comment:
`By now 12,000 Celo Gold should have been sent to your Validator and Validator Group account addresses. This will allow you to submit transactions to the network via the `celocli`. To do so, you'll need to have a running node with access to the account keys. Start one by running the following commands on your local machine:

COMANDS HERE


Once the node is synced, we can register our accounts:`